src/Illuminate/Database/Query/Builder.php

```diff
@@ -454,7 +454,7 @@ public function where($column, $operator = null, $value = null, $boolean = 'and'
 		}
 		elseif ($this->invalidOperatorAndValue($operator, $value))
 		{
-			throw new InvalidArgumentException("Value must be provided.");
+			throw new InvalidArgumentException("Illegal operator and value combination.");
```

Comment:
`Illegal` sounds a bit harsh. What about `Invalid operator and value combination.`?